estate_account/__manifest__.py

```diff
@@ -0,0 +1,9 @@
+{
+    'name': "Real Estate Account",
+    'summary': "Estate Account Module helps to create invoice.",
+    'application': True,
+    'category': "Tutorials",
+    'installable': True,
+    'depends': ['account', 'base', 'estate'],
```

Comment:
```suggestion
    'depends': ['account', 'estate'],
```
Nice starting with bridge module 😄, same comment as previous minimal dependents